docs/source/getting-started/app-architecture.rst

```diff
@@ -48,14 +48,15 @@ The signatories are the parties whose authorization is required to create or arc
 The observers are the parties who are able to view the contract on the ledger.
 In this case all users that a particular user is following are able to see the user contract.
 
+A note on naming is also in order here. Each party has both a unique random string that defines that party, and a human-readable user name that is used at login. In addition, a user can set a display name or alias that is visible to other users. Only the alias can be changed. Clicking on a contract will show you the party’s unique string, not the user name or alias. 
```

Comment:
The clicking on a contract part applies to Navigator but this guide isn’t using navigator at all.